code_samples/data_migration/config/custom_services.yaml

```diff
@@ -7,3 +7,11 @@ services:
     App\Migrations\Action\AssignSectionExecutor:
         tags:
             - { name: 'ibexa.migrations.executor.action.content', key: !php/const App\Migrations\Action\AssignSection::TYPE }
+
+    App\Migrations\Matcher\SectionIdentifierNormalizer:
+        tags:
+            - { name: 'ibexa.migrations.serializer.normalizer', priority: 10 }
```

Comment:
I think that all of our built-in normalizers are set to a negative value in `priority`, so user defined normalizers always go first unless specifically needed (and adjusted via `priority`). So it's cool to have the information that this can be used, but it's should not be needed.

If that's not the case then it's a bug.